README.md

```diff
@@ -103,6 +103,16 @@ The `ChatMember` object now has two new fields:
 - `joined_date`: integer, unix timestamp, when has the user joined
 - `inviter`: `User`, the inviter
 
+#### Object `Chat`
+The `Chat` object now has two new fields:
+- `is_verified`: bool, default false. Is the chat verified by Telegram, clients show a verified batch
```

Comment:
I think you should clarify that the new fields are _optional_ and therefor may not be present

Reply:
Okay, clarified that they are optional